docs/data/base/components/tabs/tabs.md

```diff
@@ -67,6 +67,12 @@ The next demo shows how to apply custom styles to a set of tabs:
 
 {{"demo": "UnstyledTabsCustomized.js"}}
 
+### Vertical
```

Comment:
Can you move this section down to underneath the `## Customization` header? It can be the first item, above the section about TypeScript.